css/types/color.json

```diff
@@ -82,6 +82,39 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "mixed_type_parameters": {
+            "__compat": {
+              "description": "Mix <code>&lt;percentage&gt;</code> and <code>&lt;number&gt;</code> in parameters",
```

Comment:
I wanted to mirror existing sub-features like `alpha parameter` and `float parameters`.

Suggestions for better naming are more than welcome!

Reply:
Maybe `mixed` is not the best word.
Because some notations like `hsl` and `hwb` have always had mixed typings.